### DISCUSSION TITLE
Zoom hotkey

### DISCUSSION THREAD

**Discussion by janw-me** [standardnotes/forum#3534] [Suggestions]
Most applications use `ctrl -` to zoom out, the Standard Notes also use this.
To zoom in `ctrl +` works also in pretty much every application, including Standard Notes.

But because the `+` character is "under" the `=` sign the actual key combo pressed is `ctrl shift =`.
Because this is an extra key press most applications take out the middle man and also map `ctrl =` to zoom in.
All Browsers, slack(also electron), VSCode support both

But Standard notes only works with `ctrl shift =`
So the actual request:

**TLDR:**
Can `ctrl =`  also be mapped to zoom-in in the desktop app?
